doc/api/http.md

```diff
@@ -587,9 +587,11 @@ added: v1.6.0
 -->
 
 * `name` {string}
-* Returns: {string}
+* Returns: {string, string[]} - type as set via `setHeader()`
 
 Reads out a header on the request. Note that the name is case insensitive.
+The return value is that one set via `setHeader()` as coercing to string
+happens during transmission.
```

Comment:
This sentence is a bit difficult to understand. How about this instead: 

```markdown
The value of the return type depends on the arguments provided to `setHeader()`.
```

Also, `setHeaders()` should be a link to that function in the docs.

Reply:
@Trott `The value of the return type` -> `The type of the return value`?

Reply:
@vsemozhetbyt Whoops, yes, thanks for the correction!